codeSnippets/snippets/tutorial-full-stack-task-manager/integration-test/build.gradle.kts

```
@@ -0,0 +1,44 @@
+plugins {
+    alias(libs.plugins.androidLibrary)
+    alias(libs.plugins.kotlinAndroid)
+    alias(libs.plugins.kotlinPluginSerialization)
+}
+
+dependencies {
+    implementation(projects.shared)
+    implementation(projects.server)
+    implementation(projects.composeApp)
+
+    implementation(libs.ktor.server.content.negotiation.jvm)
+    implementation(libs.ktor.client.content.negotiation)
+    implementation(libs.ktor.serialization.kotlinx.json)
+    implementation(libs.androidx.core.ktx)
+
+    testImplementation(libs.ktor.server.test.host)
+    testImplementation(libs.kotlin.test)
+    testImplementation(libs.kotlin.test.junit5)
+
+    testImplementation(libs.junit.jupiter.api)
+    testRuntimeOnly(libs.junit.jupiter.engine)
+}
+
+android {
+    namespace = "io.github.kez_lab.multipatform.full.integrationtest"
```

Let's keep a namespace similar to the one used in `composeApp` module:
```suggestion
    namespace = "org.example.ktor.integrationtest"
```